waiter/test/waiter/process_request_test.clj

```diff
@@ -391,6 +391,36 @@
           {:keys [status]} (handler request)]
       (is (= http-200-ok status)))))
 
+(deftest test-wrap-maintenance-mode
+  (testing "returns 503 for token in maintenance mode with custom message"
+    (let [handler (wrap-maintenance-mode (fn [_] {:status http-200-ok}))
+          request {:waiter-discovery {:token-metadata {"maintenance" {"message" "test maintenance message"}}
```

Comment:
We can extract `"test maintenance message"` into a constant and refer to the constant in the assertion below.